classes/Http/Controller/Reviewer/SpeakersController.php

```diff
@@ -27,7 +27,17 @@ public function indexAction(Request $req)
         // Set up our page stuff
         $pagerfanta = new Pagination($speakers);
         $pagerfanta->setCurrentPage($req->get('page'));
-        $pagination = $pagerfanta->createView('/reviewer/speakers?');
+        
+        // Paginator url
+        if (!empty($req->get('order_by'))) {
+            $pagination = $pagerfanta->createView('/reviewer/speakers?search='. $search .'&order_by='. $order_by. '&order='. $order .'&');
+        }
+        else if (!empty($req->get('search'))){
```

Comment:
Coding standard violations (PSR 1/2):
- `elseif` instead of `else if`
- `else/elseif` should be at the same line as `if`s closing bracket

Reply:
Updated PSR-2